projects/igniteui-angular/src/lib/services/exporter-common/base-export-service.ts

```diff
@@ -805,50 +825,76 @@ export abstract class IgxBaseExporter {
         }
     }
 
-    private getColumns(columns: IgxColumnComponent[]): IColumnList {
+    private getColumns(columns: any[]): IColumnList {
```

Comment:
There is no reason to use `any`.
```suggestion
    private getColumns(columns: IgxColumnComponent[]): IColumnList {
```